polytope_server/common/staging/s3_staging.py

```diff
@@ -47,15 +47,16 @@ def __init__(self, config):
         access_key = config.get("access_key", "")
         secret_key = config.get("secret_key", "")
         self.bucket = config.get("bucket", "default")
+        use_secure = config.get("use_secure", False) == True
```

Comment:
The name could be just secure to remain consistent with the other parameters

Reply:
9566e33